SU-HSDO/suhumsci
# SHS-4875: regression - hb-main-body-image-detail no longer wraps text next to image - in Chrome only

# READY FOR REVIEW

## Summary
- Fix regression where floated images specifically added through layout build main region were no longer floated and wrapping text.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. Verify the original reported issue is still fixed. Go to `/events/colloquium-series` and verify the floated image next to "Colloquium Series" is no longer collapsing the view below it.
2. Spin up the "Gender" site and go to https://gender.suhumsci.loc/people/adrian-daub - verify the image for Adrian Duab is floated right and the text below it is properly wrapping around the image. 
![image (14)](https://user-images.githubusercontent.com/369018/226467088-2620c035-d154-4a4c-95ae-484d2d34fd3d.png)

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
